.travis.yml

```diff
@@ -3,28 +3,28 @@ go:
 - 1.13.x
```

Comment:
we should definitely drop 1.13 now. 1.16 is out and only 1.16 and 1.15 will be getting fixes now.

Reply:
Good idea. Dropped 1.13, looks like there's some minor work to get 1.16 working so I'll leave that off for now.